src/artkit/model/llm/openai/_openai.py

```diff
@@ -103,27 +104,63 @@ async def get_response(
         message: str,
         *,
         history: ChatHistory | None = None,
+        streaming: bool = False,  # streaming argument
```

Comment:
As discussed on Slack, let's avoid adding `streaming` to the function signature. 

Instead, we can do this:

```python
async def get_response(
        self,
        message: str,
        *,
        history: ChatHistory | None = None,
        **model_params: dict[str, Any],
    ) -> list[str]:
    streaming = model_params.get("streaming", False)
```

In the code above, if `streaming` was passed in as an optional argument, then it will be part of the `model_params` dict. The last line in the code block searches for `streaming` in `model_params`, and returns its value (True or False) if it is given, otherwise if it is not in the dictionary, we get the default value `False`.

Reply:
Done